apps/web-giddh/src/app/new-inventory/component/adjust-inventory-list/adjust-inventory-list.component.html

```diff
@@ -1,52 +1,347 @@
+                            </th>
+                            <td mat-cell *matCellDef="let element">
+                                {{ element.refNo }}
+                            </td>
+                        </ng-container>
+
+                        <!-- Name Column -->
+                        <ng-container matColumnDef="name">
+                            <th
+                                width="20%"
+                                mat-header-cell
+                                *matHeaderCellDef
+                                #searchNameContainer
+                                class="td-search-box"
+                                (clickOutside)="handleClickOutside($event, searchNameContainer, localeData?.stock_name)"
+                            >
+                                <ng-container
+                                    *ngTemplateOutlet="
+                                        searchTemplate;
+                                        context: {
+                                            $implicit: showName,
+                                            fieldName: localeData?.stock_name,
+                                            title: localeData?.stock_name,
+                                            formControl: adjustInventoryListForm.controls['name']
+                                        }
+                                    "
+                                ></ng-container>
+                            </th>
+                            <td mat-cell *matCellDef="let element">{{ element.adjustedProduct?.name }}</td>
+                        </ng-container>
+
+                        <!-- Reason Column -->
+                        <ng-container matColumnDef="reason">
+                            <th
+                                width="20%"
+                                mat-header-cell
+                                *matHeaderCellDef
+                                #searchReasonContainer
+                                class="td-search-box"
+                                (clickOutside)="handleClickOutside($event, searchReasonContainer, localeData?.reason)"
+                            >
+                                <ng-container
+                                    *ngTemplateOutlet="
+                                        searchTemplate;
+                                        context: {
+                                            $implicit: showReason,
+                                            fieldName: localeData?.reason,
+                                            title: localeData?.reason,
+                                            formControl: adjustInventoryListForm.controls['reason']
+                                        }
+                                    "
+                                ></ng-container>
+                            </th>
+                            <td mat-cell *matCellDef="let element">{{ element.reason?.name }}</td>
```

Comment:
```suggestion
                            <td mat-cell *matCellDef="let element">{{ element?.reason?.name }}</td>
```

change in all columns